src/spyglass/spikesorting/v0/spikesorting_burst.py

```diff
@@ -0,0 +1,393 @@
+from itertools import permutations
+from typing import Dict, List, Tuple
+
+import datajoint as dj
+import numpy as np
+from scipy import stats
+from spikeinterface.postprocessing.correlograms import (
+    WaveformExtractor,
+    compute_correlograms,
+)
+
+from spyglass.settings import test_mode
+from spyglass.spikesorting.utils_burst import (
+    calculate_ca,
+    calculate_isi_violation,
+    plot_burst_by_sort_group,
+    plot_burst_metrics,
+    plot_burst_pair_peaks,
+    plot_burst_peak_over_time,
+    plot_burst_xcorrel,
+    validate_pairs,
+)
+from spyglass.spikesorting.v0.spikesorting_curation import (
+    CuratedSpikeSorting,
+    Curation,
+    Waveforms,
+)
+from spyglass.utils import SpyglassMixin, logger
+from spyglass.utils.waveforms import _get_peak_amplitude
+
+schema = dj.schema("spikesorting_burst_v0")
+
+
+@schema
+class BurstPairParams(SpyglassMixin, dj.Lookup):
+    """Parameters for burst pair selection
+
+    burst_params_name: name of the parameter set
+    params: dictionary of parameters, including:
+        sorter: spike sorter name
+        correl_window_ms: window for cross-correlogram in ms
+        correl_bin_ms: bin size for cross-correlogram in ms
+        correl_method: method for cross-correlogram calculation
+    """
+
+    definition = """
+    burst_params_name: varchar(32) # name of the parameter set
+    ---
+    params: blob # dictionary of parameters
+    """
+    contents = [
+        (
+            "default",
+            dict(
+                sorter="mountainsort4",
+                correl_window_ms=100.0,
+                correl_bin_ms=5.0,
+                correl_method="numba",
+            ),
+        )
+    ]
+
+    def get_params(self, key: dict) -> dict:
+        """Given a key with burst_params_name, return the parameters"""
+        return self.get_params_blob_from_key(key)
+
+
+@schema
+class BurstPairSelection(SpyglassMixin, dj.Manual):
+    definition = """
+    -> CuratedSpikeSorting
+    -> BurstPairParams
+    """
+
+    def insert_by_sort_group_ids(
+        self,
+        nwb_file_name: str,
+        session_name: str,
+        sort_group_ids: List[int] = None,
+        sorter: str = "mountainsort4",
+        curation_id: int = 1,
```

Comment:
why does this default to curation_id = 1? It might be better to let the user specify so that it isn't assigned to the wrong curation.